### DISCUSSION TITLE
Windows - PowerShell best practice to install manually stay up to date

### DISCUSSION THREAD

**Discussion by Ergamon** [JanDeDobbeleer/oh-my-posh#2172] [Troubleshoot]
Sadly you removed the module and now I am asking myself how to migrate.

Usually I keep my PowerShell setup up to date by running 
Get-InstalledModule | Update-Module

Now oh-my-posh is more or less only an external application and keeping stuff up to date has to be self organized. As modules are part of the local user profile, staying up to date was quite simple.

In the documentation for Windows you only describe setting up and updating by package managers. winget is not available on windows servers and installing a package manager just for oh-my-posh feels ridiculous.

So what is the correct way to install manually on windows?

Perhaps you can offer a script that basically does what the module has done before for updating? Personally I had never any problem with the oh-my-posh module fetching the executable in the background

**Comment by JanDeDobbeleer** [comment]
I like the idea of a simple way to download the installer and install. That's a good idea for these use-cases. Will create something this evening.

**Reply by JanDeDobbeleer** [reply]
@Ergamon you will soon be able to use the following command:

```powershell
Set-ExecutionPolicy Bypass -Scope Process -Force; Invoke-Expression ((New-Object System.Net.WebClient).DownloadString('https://ohmyposh.dev/install.ps1'))
```
